network.go

```diff
@@ -1,15 +1,28 @@
 package main
 
-//func GetPublicIPAddress() {
+import "net"
+
+//func GetPublicIPAddress() string {
 //	//Implement the function to return the public address of the pc.
 //	//Do not forget to handle errors if the pc isn't connected to a public network!
 //	// You will need to return a String.
 //}
 //
-//func GetLocalIPAddress() {
-//	//Implement to function to return the local IP Address of the pc.
-//	// You will need to return a String.
-//}
+func GetLocalIPAddress() {
+    addrs, err := net.InterfaceAddrs()
+    if err != nil {
+        StandardPrinter(ErrorRedColor, "Could not retrieve your local ip")
+    }
+    for _, address := range addrs {
+        // check the address type and if it is not a loopback the display it
+        if ipnet, ok := address.(*net.IPNet); ok && !ipnet.IP.IsLoopback() {
+            if ipnet.IP.To4() != nil {
+		    ResultPrinter("Local ip addres: ", ipnet.IP.String())
```

Comment:
Please correct the spelling errors and make sure IP in the result is in capital letters